rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -476,6 +476,7 @@ def _decode_curve_gauge_events(self, context: DecoderContext) -> DecodingOutput:
         provider = hex_or_bytes_to_address(context.tx_log.topics[1])
         gauge_address = context.tx_log.address
         raw_amount = hex_or_bytes_to_int(context.tx_log.data)
+        found_relevant_event = False
```

Comment:
same comment as before. We should use good names for the variables since someone that is not you or me will see this name and won't know for what this variable has found relevant events